src/Persistence/Wolverine.Marten/IMartenOp.cs

```diff
@@ -225,10 +242,25 @@ public void Execute(IDocumentSession session)
 }
 
 public class StoreDoc<T> : DocumentOp where T : notnull
+{
+    private readonly T _document;
+
+    public StoreDoc(T document) : base(document)
+    {
+        _document = document;
+    }
+
+    public override void Execute(IDocumentSession session)
+    {
+        session.Store(_document);
+    }
+}
+
+public class StoreManyDocs<T> : DocumentsOp where T : notnull
 {
     private readonly T[] _documents;
 
-    public StoreDoc(params T[] documents) : base(documents)
+    public StoreManyDocs(T[] documents) : base(documents)
```

Comment:
https://github.com/JasperFx/wolverine/issues/1162#issuecomment-2530791285

Reply:
Missed that! Fixed it